### PR TITLE
[SMP-1345] SMP 0.8.0 release notes

### PR DESCRIPTION
**Pull request by brian-f-harness**
SMP 0.8.0 release notes

For reviewers, previews are available:

[Self-Managed Enterprise Edition NG](https://64c8968189bb8b3c60141902--harness-developer.netlify.app/release-notes/self-managed-enterprise-edition)

[Harness SaaS Release Notes (FirstGen)](https://64c8968189bb8b3c60141902--harness-developer.netlify.app/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes)

[What's new](https://64c8968189bb8b3c60141902--harness-developer.netlify.app/release-notes/whats-new)

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by pratmit — APPROVED** [review]
LGTM and approved based on merging these changes.

File: release-notes/self-managed-enterprise-edition.md [hunk 1]
```diff
@@ -17,19 +17,636 @@ import delete_project from './static/delete-project.png'
...
 
-## Latest - June 30, 2023, version 79421
+## Latest - July 31, 2023, version 79819
 
-### Known issue
+### Known issues
 
 -  Terraform-based pipeline automation does not function correctly in Istio environments. UI operations are not affected. (PL-39776)
 
+- The `log-service` created separate Redis streams and set the expiration of all keys. Harness has temporarily disabled Redis in the `log-service` for High availability configuration. (CI-9000)
```
Comment (akshay-khole) [inline release-notes/self-managed-enterprise-edition.md]: I would say - The `log-service` created separate Redis streams and set the expiration of all keys. Harness has temporarily does not support High availability configuration for the `log-service` till this issue is resolved

File: release-notes/self-managed-enterprise-edition.md [hunk 2]
```diff
@@ -17,19 +17,636 @@ import delete_project from './static/delete-project.png'
...
+| LE NG | 68004 |
+
+#### Self-Managed Enterprise Edition
+
+- Harness now supports external self-managed databases for high availability. (SMP-1577, SMP-1617, SMP-1645, SMP-1646)
+
+  To learn how to configure an external self-managed database, go to the following tutorials.
+
+<!---
+  - [MongoDB](/tutorials/self-managed-enterprise-edition/use-an-external-self-managed-mongodb/)
```
Comment (akshay-khole) [inline release-notes/self-managed-enterprise-edition.md:56]: will these URLs be replaced ?

File: release-notes/self-managed-enterprise-edition.md (hunk 1 above)
Comment (brian-f-harness) [reply release-notes/self-managed-enterprise-edition.md]: @akshay-khole @pratmit added a commit to address. Thank you both for your review.

File: release-notes/whats-new.md [hunk 3]
```diff
@@ -19,7 +19,20 @@ Review the notes below to learn about the new features that are Generally Availa
...
-## Latest - July 28, 2023
+## July 31, 2023
+
+### Self-Managed Enterprise Edition, version 79819
+
+- Harness now supports external self-managed databases for high availability. (SMP-1577, SMP-1617, SMP-1645, SMP-1646)
+
+  To learn how to configure an external self-managed database, go to the following tutorials.
+
+  - [MongoDB](/tutorials/self-managed-enterprise-edition/use-an-external-self-managed-mongodb/)
```
Comment (akshay-khole) [inline release-notes/whats-new.md:30]: will these URLs be replaced ?

File: release-notes/self-managed-enterprise-edition.md [hunk 4]
```diff
@@ -23,7 +23,7 @@ Review the notes below for details about recent changes to Harness Self-Managed
 
 -  Terraform-based pipeline automation does not function correctly in Istio environments. UI operations are not affected. (PL-39776)
 
-- The `log-service` created separate Redis streams and set the expiration of all keys. Harness has temporarily disabled Redis in the `log-service` for high availability configurations. (CI-9000)
+- The `log-service` created separate Redis streams and set the expiration of all keys. Harness temporarily does note support high availability configuration for the `log-service` until this issue is resolved. (CI-9000)
```
Comment (akshay-khole) [inline release-notes/self-managed-enterprise-edition.md:26]: note -> not

Comment (brian-f-harness) [reply release-notes/self-managed-enterprise-edition.md:26]: thanks, @akshay-khole

File: docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md [hunk 5]
```diff
@@ -16,7 +16,74 @@ For Harness SaaS release notes, go to [Harness SaaS Release Notes](https://devel
...
+| Verification Service | 79819 |
+| UI | 79800 |
+| Learning Engine | 67903 | 
+| Gateway | 11201 |
+
+### What's new
+
+- Upgraded the delegate JRE to 11.0.19_7. (PL-37994)
+
+- Free, Community, and Paid accounts can now have a maximum of 100, 100, and 50000 users, respectively. (PL-39235)
```
Comment (aimurphy) [inline docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md:36]:
```suggestion
- Free, Community, and Paid accounts can now have a maximum of 100, 1000, and 50000 users, respectively. (PL-39235)
```

Middle number is missing 1 or more zeros.

Comment (brian-f-harness) [reply docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md:36]: @aimurphy I checked the ticket. A list may work better. The limits are Free - 100, Community - 100, and Paid - 50000

https://harness.atlassian.net/browse/PL-39235

https://developer.harness.io/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes/#june-19-2023-version-79606

Comment (aimurphy) [reply docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md:36]: oh okay.

File: release-notes/self-managed-enterprise-edition.md [hunk 6]
```diff
@@ -17,19 +17,632 @@ import delete_project from './static/delete-project.png'
...
+
+- Jenkins step marking voluntary settings as mandatory. (CDS-70071, ZD-44924)
+  
+  User were unable to save empty values for job parameters in the Jenkins step due to validations present in the UI. This has been fixed now and the incorrect validations have been removed.
+
+- Fixed an issue to honor default values in runtime inputs when trying to reconcile. (CDS-70075, ZD-44892)
+
+- Unable to trigger pipelines using custom webhook triggers because the latest enhancement to custom webhook triggers requires a unique token for authentication. (CDS-70320, ZD-45022)
+  
+  This issue was caused by introducing a new API for custom webhook triggers, `/v3`, that generates a token for authenticating webhook triggers. You can see the token when you copy the trigger cURL command in Harness. This issues is fixed by allowing users to continue to use the previous API, `v2`, when the feature flag `SPG_DISABLE_CUSTOM_WEBHOOK_V3_URL` is enabled.
```
Comment (douglas-j-bothwell) [inline release-notes/self-managed-enterprise-edition.md:266]:
```suggestion
  This issue was caused by introducing a new API for custom webhook triggers, `/v3`, that generates a token for authenticating webhook triggers. You can see the token when you copy the trigger cURL command in Harness. This issue is fixed by allowing users to continue to use the previous API, `v2`, when the feature flag `SPG_DISABLE_CUSTOM_WEBHOOK_V3_URL` is enabled.
```

File: release-notes/self-managed-enterprise-edition.md [hunk 7]
```diff
@@ -17,19 +17,632 @@ import delete_project from './static/delete-project.png'
...
+
+#### Harness Delegate
+
+- The account-level **Session Timeout (in minutes)** allowed values greater than the 4320 minute maximum. (PL-32498)
+
+   This issue has been resolved by adding a code validation. The field no longer accepts values above 4320 minutes.
+
+   This item requires Harness Delegate version 79707. For information about features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate).
+
+- Enhanced handling and logging for the `No enum constant io.harness.delegate.message.MessengerType.WATCHEIN` exception to enable the actual malformed message. This error indicates that a message is malformed and only occurs when there is an error during writing, for example, out of disk, process killed, etc. (PL-38245)
```
Comment (douglas-j-bothwell) [inline release-notes/self-managed-enterprise-edition.md:442]:
```suggestion
- Enhanced handling and logging for the `No enum constant io.harness.delegate.message.MessengerType.WATCHEIN` exception to enable the actual malformed message. This error indicates that a message is malformed and only occurs when there is a write error such as out of disk, process killed, etc. (PL-38245)
```

File: docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md [hunk 8]
```diff
@@ -16,7 +16,74 @@ For Harness SaaS release notes, go to [Harness SaaS Release Notes](https://devel
...
+
+- You can now skip steady state checks for native Helm deployments. This helps avoid Harness running unnecessary checks for resources where this is not required. This option is available in Helm Deployment and Helm Rollback steps. This feature requires delegate version 1.0.797xx or higher. (CDS-70124)
+
+### Early access
+
+This release does not include any early access features.
+
+### Fixed issues
+
+- Helm delegate installation failed in Self-Managed Enterprise Edition. (PL-39028)
```
Comment (douglas-j-bothwell) [inline docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md:48]:
```suggestion
- Helm delegate installation failed in Self-Managed Enterprise Edition.  This issue is fixed with a code enhancement. The `DELPOY_MODE` is now set to `KUBERNETES_ONPREM` for Self-Managed Enterprise Edition Docker and Helm delegates. (PL-39028)
```

File: docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md [hunk 9]
```diff
@@ -16,7 +16,74 @@ For Harness SaaS release notes, go to [Harness SaaS Release Notes](https://devel
...
+
+This issue is fixed with a code enhancement. The `DELPOY_MODE` is now set to `KUBERNETES_ONPREM` for Self-Managed Enterprise Edition Docker and Helm delegates.
```
Comment (douglas-j-bothwell) [inline docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md:50]:
```suggestion

```

File: docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md [hunk 10]
```diff
@@ -16,7 +16,74 @@ For Harness SaaS release notes, go to [Harness SaaS Release Notes](https://devel
...
+  The environment variables for these timeouts are:
+
+  - Read timeout: `K8S_API_CLIENT_READ_TIMEOUT`
+  - Connect timeout: `K8S_API_CLIENT_CONNECT_TIMEOUT`
+
+- The delegate connectivity capability check for Azure Web App deployments failed. (CDS-71432, ZD-44862)
+
+   This issue is fixed by using task category mapping to assign the Azure App Services task type deployments to specific delegates.
+
+- (CDS-74584)
```
Comment (douglas-j-bothwell) [inline docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md:83]:
```suggestion

```

**Review by douglas-j-bothwell — APPROVED** [review]

**Review by deepakbolangady — APPROVED** [review]
LGTM

**Pull request merged**